docs/output.md

```diff
@@ -37,32 +100,104 @@ The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes d
+### Pairs
 
-### Pipeline information
+<details markdown="1">
+<summary>Output files</summary>
+
+* `pairs/`
+    * `raw/*`: The raw reads pairs for each sample and stats for the pairs.
+    * `filtered/*`: The filtered files. The unselected.pairs.gz files in samefrag folder can be used to generate virtual 4C plots.
+    * `QC/*`: The quality analysis results.
+
+</details>
+
+The raw and filtered pairs exported by [pairtools](https://pairtools.readthedocs.io/en/latest/).
+The quality analysis for filtered pairs were done by [pairsqc](https://github.com/4dn-dcic/pairsqc).
+
+### Cooler
 
 <details markdown="1">
 <summary>Output files</summary>
 
-* `pipeline_info/`
-    * Reports generated by Nextflow: `execution_report.html`, `execution_timeline.html`, `execution_trace.txt` and `pipeline_dag.dot`/`pipeline_dag.svg`.
-    * Reports generated by the pipeline: `pipeline_report.html`, `pipeline_report.txt` and `software_versions.tsv`.
-    * Reformatted samplesheet files used as input to the pipeline: `samplesheet.valid.csv`.
+* `cooler/mcool/*`: The mcool files for each group.
+* `cooler/hic/*`: The .hic files for each group.
 
 </details>
 
-[Nextflow](https://www.nextflow.io/docs/latest/tracing.html) provides excellent functionality for generating various reports relevant to the running and execution of the pipeline. This will allow you to troubleshoot errors with the running of the pipeline, and also provide you with other information such as launch commands, run times and resource usage.
+[Cooler](https://cooler.readthedocs.io/en/latest/index.html) files for visualization.
+To view the `.mcool` files, you may want to try [higlass-manage](https://github.com/higlass/higlass-manage).
+Once the `higlass-manage` is install, view the files by
+
+```bash
+higlass-manage view cooler/mcool/your.mcool
+```
+
+[Juicebox](https://github.com/aidenlab/Juicebox) can be used to view the `.hic` file.
+`.hic` file is the second choices for visualization in case installation of `higlass-manage` is impossible.
+
+### Call peaks for short reads
+
+<details markdown="1">
+<summary>Output files</summary>
+
+* `ATACpeak/`
+    * `R2_bigwig/*`: The bigWig files of R2_reads.
+    * `peaks_per_Group/*`: The called peaks for each group.
+    * `merged_peaks/*`: The merged peaks for all groups.
+
+</details>
+
+The peaks called ATAC reads (R2 reads) by [MACS2](https://github.com/macs3-project/MACS).
+
+### TADs and loops
+
+<details markdown="1">
+<summary>Output files</summary>
+
+* `MAPSpeak/`
+    * `bin*/*`: The annotated TADs and loops.
+    * `*`: The tables of TADs and loops with p-value and fdr.
+
+</details>
+
+The TADs and loops are called by [MAPS](https://pubmed.ncbi.nlm.nih.gov/30986246/).
+MAPS is a model-based analysis of long-range chromatin interactions.
```

Comment:
https://github.com/nf-core/hicar/pull/1#discussion_r741483454

If it's possible to add a little more content to this documentation then that would be great. Pretend that you're describing the outputs to someone totally new to this data type and analysis type.

For an example of another pipeline with really nice docs in this file, check out the atacseq pipeline: https://nf-co.re/atacseq/dev/output#call-peaks

Reply:
I will try to add more detailed descriptions.